src/main/resources/css/chapters.css

```diff
@@ -32,6 +32,15 @@
     height: 24px;
     width: 24px;
 }
+.chaptercard:hover {
+    background: #b0b0b029 !important
+}
+.chaptercard.unavailable {
+    overflow: hidden;
+}
+.l-list-container > div:last-child a.chaptercard {
```

Comment:
I think can we move this to common.css and have this be consistent between the languages, books and chapter lists? `.l-list-container > div:last-child a { `?

Reply:
sure, let me fix that